### PR TITLE
Nana Test 2

### PR DESCRIPTION
**Pull request by NatashaBaiq**
Try again and again

### CODE REVIEW

**Comment by NatashaBaiq** [comment]
Please look at my 'nanatest' @alan-namoos :heart: :heart: :heart:

**Comment by Alan-Namoos** [comment]
I think it looks cool @NatashaBaiq  I will accept it, but I don't know how 😛

File: nanatest [hunk 1]
```diff
@@ -6,5 +6,6 @@
   </head>
   <body>
   <h1>GitHub Nana Test<h1>
+  <h1>GitHub Nana Test 2<h1>
```
Comment (Alan-Namoos) [inline nanatest:9]: Here you should use paragraph @NatashaBaiq

Comment (NatashaBaiq) [reply nanatest:9]: here here... how? what? ha? :laughing: :rofl:

**Pull request merged**